scripts/cmake/ClangSanitizer.cmake

```diff
@@ -0,0 +1,19 @@
+if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "3.6")
+	return()
+endif()
+
+option(OGS_ADDRESS_SANITIZER OFF "Use Clangs AddressSanitizer")
+option(OGS_UNDEFINED_BEHAVIOR_SANITIZER OFF "Use Clangs UndefinedBehaviorSanitizer")
+
+if(OGS_ADDRESS_SANITIZER)
+	set(SANITIZE_FLAG_VALUE "address")
+	set(ADDITIONAL_FLAGS "-fno-omit-frame-pointer")
+endif()
+
+if(OGS_UNDEFINED_BEHAVIOR_SANITIZER)
+	set(SANITIZE_FLAG_VALUE "${SANITIZE_FLAG_VALUE},undefined")
```

Comment:
I'm not sure if both sanitizers can be use at the same time. Did not find anything on the [clang docu](http://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html) but usually they specify if sanitizers can be combined (like the [LeakSanitizer](http://clang.llvm.org/docs/LeakSanitizer.html)).